packages/monorepo.cspell/cspell.config.js

```diff
@@ -143,6 +143,9 @@ const cspell = {
 
         // Used in a stub
         "Clooney",
+
+        // Used in API docs
+        "AKIAIOSFODNN",
```

Comment:
Added this override for now, I feel like maybe we should rethink spell check on these autogenerated things, but just need to push this through for now